climada_petals/test/test_wildfire.py

```diff
@@ -84,11 +112,35 @@ def test_hist_fire_season_firms_pass(self):
         self.assertAlmostEqual(wf.intensity[0, 8000], 0.0)
         self.assertAlmostEqual(wf.fraction.max(), 1.0)
         self.assertAlmostEqual(wf.fraction.min(), 0.0)
+        
+    def test_from_hist_fire_season_firms_pass(self):
+        """ Test set_hist_event_year_set """
+        wf = WildFire()
+        wf = wf.from_hist_fire_seasons_FIRMS(TEST_FIRMS)
+
+        self.assertEqual(wf.tag.haz_type, 'WFseason')
+        self.assertEqual(wf.units, 'K')
+        self.assertTrue(np.allclose(wf.event_id, np.arange(1, 2)))
+        self.assertTrue(np.allclose(wf.date, np.array([735964])))
+        self.assertTrue(np.allclose(wf.orig, np.ones(1, bool)))
+        self.assertEqual(wf.event_name, ['2016'])
+        self.assertTrue(np.allclose(wf.frequency, np.ones(1)))
+        self.assertEqual(wf.intensity.shape, (1, 51042))
+        self.assertEqual(wf.fraction.shape, (1, 51042))
+        self.assertEqual(wf.intensity[0, :].nonzero()[1][0], 47)
+        self.assertEqual(wf.intensity[0, :].nonzero()[1][20], 1296)
+        self.assertEqual(wf.intensity[0, :].nonzero()[1][100], 4005)
+        self.assertAlmostEqual(wf.intensity[0, 47], 305.0)
+        self.assertAlmostEqual(wf.intensity[0, 1296], 342.5)
+        self.assertAlmostEqual(wf.intensity[0, 4005], 318.5)
+        self.assertAlmostEqual(wf.intensity[0, 8000], 0.0)
+        self.assertAlmostEqual(wf.fraction.max(), 1.0)
+        self.assertAlmostEqual(wf.fraction.min(), 0.0)
 
     def test_proba_fire_season_pass(self):
         """ Test probabilistic set_probabilistic_event_year_set """
         wf = WildFire()
-        wf.set_hist_fire_seasons_FIRMS(TEST_FIRMS)
+        wf = wf.from_hist_fire_seasons_FIRMS(TEST_FIRMS)
```

Comment:
Delete prior line and initalize like so:

```suggestion
        wf = WildFire.from_hist_fire_seasons_FIRMS(TEST_FIRMS)
```